features/apply_1.feature

```diff
@@ -40,5 +40,4 @@ Feature: Apply 1
   Examples:
     | # of previous applications |
     | 1                          |
-    | 2                          |
     | 3                          |
```

Comment:
And 3?

Reply:
I think that 3 should stay in, to show that 1 isn't some kind of magic number.